apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -275,6 +277,7 @@ class ScriptEditor extends React.Component {
       announcements: JSON.stringify(this.state.announcements),
       visible_to_teachers: !this.state.hidden,
       is_stable: this.state.isStable,
+      deprecated: this.state.deprecated,
```

Comment:
Looking at all the boolean props, it seems like more actually don't have 'is' in the name. I agree that it would make sense, but certainly don't see much consistency here. Worth a larger conversation/renaming of other vars?